### PR TITLE
init bizzclick prebid.js adapter

### PR DESCRIPTION
**Pull request by BizzClick**
## Type of change

- [ x] New bidder adapter   

## Description of change

- test parameters for validating bids
```
{
  bidder: 'bizzclick',
  params: {
    placementId: 'hash',
    accountId: 'bizzclickTest'
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
support@bizzclick.com
- [ x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by BizzClick** [comment]
Hey @musikele  ! Do you have chance to review my Adpater ?

**Review by musikele — CHANGES_REQUESTED** [review]
Hi there. Sorry for being late. It seems to me that the code coverage is under 80%, that is the code coverage we require for adapters. 

To test this, 

```
$ npx gulp test-coverage 

$ npx gulp view-coverage
```

File: modules/bizzclickBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+
+  /**
+   * Unpack the response from the server into a list of bids.
+   *
+   * @param {*} serverResponse A successful response from the server.
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: (serverResponse) => {
+    try {
+      serverResponse = serverResponse.body;
```
Comment (musikele) [inline modules/bizzclickBidAdapter.js:117]: Consider: 

```
if (!serverResponse || !serverResponse.body) return []
serverResponse = serverResponse.body
```

Anyway, it's generally considered a bad practice to modify the input parameter of a function. You may think of assigning it to a newly created variable.

**Review by musikele — COMMENTED** [review]
I added some notes, feel free to go into them.

**Comment by BizzClick** [comment]
Hello @musikele ! Thanks for you review! I have updated my adapter could you please check it?

**Review by musikele — APPROVED** [review]
LGTM

**Comment by musikele** [comment]
Before final merge, we need to rebase against current master, because there was an issue with a test that prevented the Continuous Integration to work effectively. 
Run these commands: 

```
git pull --rebase origin master 
git push --force 
```

This should fix the Continuous Integration.

**Comment by BizzClick** [comment]
@musikele I have made rebase to master but I still have problems with CI

**Comment by musikele** [comment]
@BizzClick this time the problem is less serious. File BizzClickBidAdapter_spec.js is not properly formatted. You may either fix all linting errors manually or run `npx gulp lint` and commit the modified files. Let me know when you're done.

**Comment by BizzClick** [comment]
Hello, @musikele  ! I would not like to bother you, but I still have CI build error after fix linting. Could you please help me resolve it?

**Comment by musikele** [comment]
@BizzClick the circleci only needed another spin. Going to merge now.

**Comment by bretg** [comment]
docs PR https://github.com/prebid/prebid.github.io/pull/2511

**Pull request merged**